labelbox/schema/model_run.py

```diff
@@ -122,6 +124,39 @@ def _wait_until_done(self, status_fn, timeout_seconds=120, sleep_time=5):
                 )
             time.sleep(sleep_time)
 
+    def upsert_predictions_and_send_to_project(
+        self,
+        name: str,
+        predictions: Union[str, Path, Iterable[Dict]],
+        project_id: str,
+        priority: Optional[int] = 5,
+    ) -> 'MEAPredictionImport':  # type: ignore
+        """ Upload predictions and create a batch import to project.
+        Args:
+            name (str): name of the AnnotationImport job as well as the name of the batch import
+            predictions (Iterable):
+                iterable of annotation rows
+            project_id (str): id of the project to import into
+            priority (int): priority of the job
+        Returns:
+            (AnnotationImport, Project)
+        """
+        kwargs = dict(client=self.client, model_run_id=self.uid, name=name)
+        project = self.client.get_project(project_id)
+        import_job = self.add_predictions(name, predictions)
+        prediction_statuses = import_job.statuses()
+        mea_to_mal_data_rows_set = set([
+            row['dataRow']['id']
+            for row in prediction_statuses
+            if row['status'] == 'SUCCESS'
+        ])
+        mea_to_mal_data_rows = list(
+            mea_to_mal_data_rows_set)[:DATAROWS_IMPORT_LIMIT]
+        batch = project.create_batch(name, mea_to_mal_data_rows, priority)
+        mal_prediction_import = Entity.MALPredictionImport.create_for_model_run_data_rows(
+            data_row_ids=mea_to_mal_data_rows, project_id=project_id, **kwargs)
+        return mea_prediction_import, batch, mal_prediction_import
```

Comment:
`mea_prediction_import` doesn't reference anything. Right now the mea import is called `import_job`